spec/domain/monitor/messages_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe Monitor::Messages do
+  before do
+    allow(GovukStatsd).to receive(:increment)
+  end
+
+  it 'increments major if routing key ends .major' do
+    expect(GovukStatsd).to receive(:increment).with("major")
+    Monitor::Messages.run('news_story.major')
```

Comment:
We shouldn't publish using the same name as the routing key. We would need to follow a similar pattern to the other monitors. Something along the lines to:

```
monitor.messages.major
```